TaroEld/BBbuilder
# Add template system

Add a template system on the init command. In the current setup, creating template can be difficult. I've done a setup which will ease up the process.

how to use/test:
- create a template in the "Templates" folder by creating a new folder in it. The name of this folder will be the name of your template. You can use placeholders "$name" or "$Name" in scripts, file name, folder name when you want them to be replaced by the name of the mod.
- run cli `bbbuilder.exe init test -replace -template [template name]`
- check that the project has been created without issue.
